gui/browsable/inc/ROOT/Browsable/RNTupleItem.hxx

```diff
@@ -0,0 +1,49 @@
+/*************************************************************************
+ * Copyright (C) 1995-2025, Rene Brun and Fons Rademakers.               *
+ * All rights reserved.                                                  *
+ *                                                                       *
+ * For the licensing terms see $ROOTSYS/LICENSE.                         *
+ * For the list of contributors see $ROOTSYS/README/CREDITS.             *
+ *************************************************************************/
+
+#ifndef ROOT7_Browsable_RNTupleItem
+#define ROOT7_Browsable_RNTupleItem
+
+#include <ROOT/Browsable/RItem.hxx>
+
+namespace ROOT {
+namespace Browsable {
+
+/** \class RNTupleItem
+\ingroup rbrowser
+\brief Representation of an RNTuple item in the browser
+\author Patryk Tymoteusz Pilichowski
+*/
+
+class RNTupleItem : public RItem {
+protected:
+   bool visualization{false}; ///< is visualization item
+public:
+   RNTupleItem() = default;
+   RNTupleItem(const std::string &_name, int _nchilds = 0, const std::string &_icon = "", bool _visualization = false)
+      : RItem(_name, _nchilds, _icon), visualization(_visualization)
+   {
+   }
+   // must be here, one needs virtual table for correct streaming of sub-classes
+   virtual ~RNTupleItem() = default;
+
+   bool IsVisualization() const { return visualization; }
+
+   virtual bool Compare(const RItem *b, const std::string &s) const override
+   {
+      auto tuple_b = dynamic_cast<const RNTupleItem *>(b);
```

Comment:
I believe that if `this->IsVisualization()` then we can automatically declare this the "winner" (meaning it should compare smaller than `b`). This is because we only expect to have 1 Visualization item and it should always be displayed before any other category. So probably you can just do:
```c++
return IsVisualization() || RItem::Compare(b, s);
```